analytics/views/support.py

```diff
@@ -277,27 +288,30 @@ def support(
             except ValidationError:
                 users.update(UserProfile.objects.filter(full_name__iexact=key_word))
 
+        plan_data: Dict[int, PlanData] = defaultdict(PlanData)
```

Comment:
Since we explicitly set `plan_data[realm.id]` for all `realm` in `realms`, the `defaultdict` could just be a normal `dict`, right?

Reply:
Updated this with a comment here and in the commit message.

Reply:
Hmm. You are correct, but that seems like a bug—we’re displaying those confirmations as if the realm doesn’t have a plan even though it does.

Reply:
Indeed. Not only that, we also display user.realm without the plan information. I expected there to be attribute errors before, but Jinja2 tolerates nonexistent attributes more than Python does, which is why this bug wasn't caught.